image/tools/lib/component/resources.sh

```diff
@@ -39,7 +39,7 @@ function backup_resource {
             echo '---' > /tmp/${type}.yaml
             for obj in $(oc get ${type} -n ${ns} | tr -s ' ' | cut -d ' ' -f 1 |  tail -n +2); do
                 echo '-' >> /tmp/${type}.yaml
-                echo "$(oc get ${type}/${obj} -n enmasse -o yaml --export | sed 's/^/  /')" >> /tmp/${type}.yaml
+                echo "$(oc get ${type}/${obj} -n ${ns} -o yaml --export | sed 's/^/  /')" >> /tmp/${type}.yaml
                 cat /tmp/${type}.yaml | gzip > ${dest}/archives/${ns}-${type}.yaml.gz
```

Comment:
There is still a bug in this part of backup. Not caused by @pb82 change, but I think we can fix it in the same PR.
We are exporting objects in a loop, but each step overrides previously exported object, so we end up just with the last one.

Reply:
should we name it `/tmp/${obj}.${type}` (or something similar)?

Reply:
@odra yeah, that's probably cleanest option. Create all the tmp files and then gzip them into ${dest}/archives/${ns}-${type}.yaml.gz together

Reply:
I think we should this pr as it is and file a new ticket/pr for this other change (so this one can be merged)

Reply:
okay, I'll create a JIRA and implement a fix